README.md

```diff
@@ -15,6 +15,7 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Preact Material Components](https://github.com/prateekbh/preact-material-components)
   - [RMWC: React Material Web Components](https://github.com/jamesmfriedman/rmwc)
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
+  - [Angular MDC](https://github.com/trimox/angular-mdc-web)
```

Comment:
Looks great! Can you insert this one line above, between lines 16 and 17 instead of lines 17 and 18?

Reply:
Appreciate the review @amrtgaber! All set.